playwright/tests/smoke/flowContainer.spec.ts

```diff
@@ -0,0 +1,148 @@
+  });
+
+  test('Flow container: 3 - checking adding and filling in an Accordion in the Restricted to 1 item Flow Container', async () => {
+    const restrictedTo1Container = page.locator(flowContainerPage.flowContainer1Item);
+    await restrictedTo1Container.click();
+    await page.click(flowContainerPage.addFlowContainerButton);
+    await page.click(flowContainerPage.imagesCheckbox);
+    await page.click(flowContainerPage.accordionCheckbox);
+    await page.click(flowContainerPage.accordionInPicker);
+    await page.click(flowContainerPage.accordionInsideRestrictedContainer);
+    await flowContainerPage
+      .typeText(flowContainerPage.accordionInsideRestrictedContainer, 'text to check', 'restricted');
+    expect.soft(await page.locator(flowContainerPage.accordionInsideRestrictedContainer)
+      .innerText()).toEqual('text to check');
+    await page.click(flowContainerPage.accordionPlusButton);
+    expect.soft(await page.locator(flowContainerPage.accordionBody).isVisible()).toBeTruthy();
+    // text is not typed without this timeout, race condition
+    await page.waitForTimeout(300);
+    await flowContainerPage
+      .typeText(flowContainerPage.accordionBody, 'accordion-body-text', '$body');
+    expect.soft(await page.locator(flowContainerPage.accordionBody).innerText()).toEqual('accordion-body-text');
+    await page.click(flowContainerPage.accordionMinusButton);
+    expect.soft(await page.locator(flowContainerPage.accordionBody).isVisible()).toBeFalsy();
+    expect.soft(await page.locator(flowContainerPage.swapComponentButton).isVisible()).toBeTruthy();
+  });
+
+  test('Flow container: 4 - checking adding and filling in a Contentful Tout in Default Width of 33%', async () => {
+    await page.click(flowContainerPage.flowContainer33Width);
+    await page.click(flowContainerPage.addFlowContainerButton);
+    await page.click(flowContainerPage.accordionCheckbox);
+    await page.click(flowContainerPage.contentfulCheckbox);
+    await Promise.all([
+      page.waitForResponse(response => response.url()
+        .includes('width_33') && response.status() === 200),
+      await page.click(flowContainerPage.accordionInPicker),
+    ]);
+    const containerWidth = await page.locator('#gatsby-focus-wrapper > div:nth-child(1) > div.my-2.container.mx-auto > div:nth-child(24) > section').boundingBox();
+    const contentfulWidth = await page.locator('#gatsby-focus-wrapper > div:nth-child(1) > div.my-2.container.mx-auto > div:nth-child(24) > section > div > div').boundingBox();
+    // @ts-ignore
+    const ratio = Math.floor((contentfulWidth.width + 40) / containerWidth.width * 100);
+    expect.soft(ratio).toBeCloseTo(32);
+  });
+
+  test('checking the added components in Preview Mode', async () => {
+    await flowContainerPage.togglePreviewMode();
+    expect.soft(await page.locator(flowContainerPage.swapComponentButton).isVisible()).toBeFalsy();
```

Comment:
@pomi ,

before checking that Add, Copy, Swap, Delete are not available in Preview Mode, do a click on any added component - to prove that Local Context Menu with that buttons doesn't show up.
Originally that buttons were available in Editor Menu, and not in Local Context Menu. that's why the 'click' step is not in the original test cases